released/packages/coq-deriving/coq-deriving.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name: "coq-deriving"
+version: "0.1.0"
+maintainer: "Arthur Azevedo de Amorim <arthur.aa@gmail.com>"
+
+homepage: "https://github.com/arthuraa/deriving"
+bug-reports: "https://github.com/arthuraa/deriving/issues"
+dev-repo: "git+https://github.com/arthuraa/deriving.git"
+license: "MIT"
+
+build: [ make "-j" "%{jobs}%" "test" {with-test} ]
+install: [ make "install" ]
+depends: [
+  "coq" { (>= "8.11" & < "8.14~") | (= "dev") }
+  "coq-mathcomp-ssreflect" {>= "1.11" | (= "dev")}
+]
+
+tags: [
+  "keyword:generic programming"
+  "category:Computer Science/Data Types and Data Structures"
+  "logpath:deriving"
+]
+authors: [
+  "Arthur Azevedo de Amorim"
+]
+
+synopsis: "Generic instances of MathComp classes"
+description: """
+Deriving provides generic instances of MathComp classes for
+inductive data types.  It includes native support for eqType,
+choiceType, countType and finType instances, and it allows users to
+define their own instances for other classes.
+"""
+
+url {
+  src: "git+https://github.com/arthuraa/deriving/archive/v0.1.0.tar.gz"
```

Comment:
This looks dubious and would explain why you get

    The sources of the following couldn't be obtained, aborting:
          - coq-deriving.0.1.0:
            Invalid_argument("VC pull_url doesn't allow checksums")

Reply:
Oops -- paste error! Hopefully it should work now... Thanks.